finagle-redis/src/main/scala/com/twitter/finagle/redis/protocol/Command.scala

```diff
@@ -141,6 +141,16 @@ object Commands {
   val PFCOUNT           = "PFCOUNT"
   val PFMERGE           = "PFMERGE"
 
+  // Scripts
+  val EVAL              = "EVAL"
+  val EVALSHA           = "EVALSHA"
+  val SCRIPT            = "SCRIPT"
+  val SCRIPTEXISTS      = SCRIPT + " EXISTS"
```

Comment:
I'm a little dubious that this makes it easier to read.  Maybe just keep it "SCRIPT EXISTS", here and below.